.gitignore

```diff
@@ -126,7 +126,7 @@ Thumbs.db
 # python stuff
 .pyprojectx
 **/__pycache__
-/.venv/
+/.venv*/
```

Comment:
what are you using to create your venvs? i just realized mine (created with uv) contains a `.gitignore` with `*` in it which causes it to be ignored already so this shouldn't even be needed

<img width="345" height="179" alt="Image" src="https://github.com/user-attachments/assets/e2c831c9-1316-4b0d-8350-8c71705722e2" />


```suggestion
```

also worth noting that there's quite a bit of project config files (mainly in `.vscode`) that assumes the venv is always located at `.venv` so you may run into issues with your setup if you use vscode